src/utils/whitelistApis.ts

```diff
@@ -3532,5 +3560,9 @@ export const API_LIST = {
             '/proxies/v8/content/aggregation/search',
             '/proxies/v8/workflow/v2/userWFApplicationFieldsSearch',
             '/proxies/v8/ratings/v1/topReviews/:orgId',
+            '/proxies/v8/demand/create',
+            '/proxies/v8/demand/read/:do_id',
+            '/proxies/v8/demand/delete/:do_id',
+            '/proxies/v8/demand/filter'
```

Comment:
update the path as follows 

'/proxies/v8/demand/content/create',
'/proxies/v8/demand/content/read/:do_id',
'/proxies/v8/demand/content/delete/:do_id',
'/proxies/v8/demand/content/search'